engine/schema/src/main/resources/META-INF/db/schema-41600to41610.sql

```diff
@@ -20,3 +20,8 @@
 --;
 
 ALTER TABLE `cloud`.`vm_work_job` ADD COLUMN `secondary_object` char(100) COMMENT 'any additional item that must be checked during queueing' AFTER `vm_instance_id`;
+
+-- Add support for VMware 7.0.2.0
+INSERT IGNORE INTO `cloud`.`hypervisor_capabilities` (uuid, hypervisor_type, hypervisor_version, max_guests_limit, security_group_enabled, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported, vm_snapshot_enabled) values (UUID(), 'VMware', '7.0.2.0', 1024, 0, 59, 64, 1, 1);
```

Comment:
It seems 7.0.3 is also out, at least I see https://customerconnect.vmware.com/en/downloads/details?downloadGroup=VC70U3A&productId=974&rPId=81436 - should we add that support too @weizhouapache @andrijapanicsb @NuxRo @alexandremattioli ?

Reply:
> It seems 7.0.3 is also out, at least I see https://customerconnect.vmware.com/en/downloads/details?downloadGroup=VC70U3A&productId=974&rPId=81436 - should we add that support too @weizhouapache @andrijapanicsb @NuxRo @alexandremattioli ?

I think, it would be good to support VMware 7.0.3 as well if not much effort is needed.

Reply:
@rohityadavcloud @sureshanaparti 
ESXi 7.0.3 has some critical issues, vmware has removed it from download site.
https://blogs.vmware.com/vsphere/2021/11/important-information-on-esxi-7-update-3.html
https://kb.vmware.com/s/article/86398

should we support it for now or in the future ?

Reply:
@weizhouapache in that case, may be in future release.